command/expand.go

```diff
@@ -89,3 +90,25 @@ func expandSources(
 
 	return ch
 }
+
+// raw source returns the only object with given path.
+func rawSource(srcurl *url.URL, followSymlinks bool) <-chan *storage.Object {
+	ch := make(chan *storage.Object, 1)
+	if storage.ShouldProcessUrl(srcurl, followSymlinks) {
+		ch <- &storage.Object{URL: srcurl}
+	}
+	close(ch)
+	return ch
+}
+
+// raw source returns the only object with given path.
+func rawSourceUrls(srcurls []*url.URL, followSymlinks bool) <-chan *storage.Object {
```

Comment:
Let's try merging `rawSource` and `rawSources` functions into a single variadic function.

```suggestion
func rawSource(followSymlinks bool, srcurls ...*url.URL) <-chan *storage.Object {
```

We can use this function in both cp/rm commands and `expandSource`  function.

Reply:
created a new single variadic function.